packaging/darwin/amazon-cloudwatch-agent-ctl

```diff
@@ -162,8 +164,9 @@ cwa_pid() {
 cwa_config() {
      config_location="${1:-}"
      restart="${2:-}"
-     mode="${3:-}"
-     multi_config="${4:-}"
+     strict_validation="${3:-}"
```

Comment:
Should strict be the 5th instead?

Reply:
I am fine with either of these since my habit of grouping boolean things together.